CONTRIBUTING.rst

```diff
@@ -169,6 +169,13 @@ Pull Requests Approval Process
 
    PR Approval Process
 
+- If the PR is only introducing minor changes which to not change the original meaning of the text or if the PR changes
+  only the files used to compile the documentation and automate the compilation
+
+  - There is only one approval is needed to get the PR merged. 
+  - The approver needs work for a different company than the creator of the PR
```

Comment:
In the light of aiming to streamline the review and change process, I would be ok with removing this requirement: if i) a change does not change the meaning of the text and ii) we have functional gates which prevent breaking changes, then I don't think there is a need for such a requirement and I am not concerned of misuse - or rather I'd prioritize speed over "potential misuse prevention".

Reply:
@gkunz when you say "removing this requirement" are you referring to the "approver needs work for a different company"?

Reply:
Yes, sorry for not clearly marking this. It is however just a suggestion. I would also be fine with the current proposal.

Reply:
@gkunz I second your proposal.

Reply:
Good edits to the PR, thanks.

Reply:
I support the proposal by @gkunz as well.  I will vote for anything which will help to create meaningful and industry relevant contents in a speedy, but organised, manner.  The industry is moving fast, and we want to be ahead, not left behind! 
So  only 1 approval (without any affiliation limitation) for PR should be required in case this PR is introducing minor changes which to not change the original meaning of the text, or if the PR changes  only the files used to compile the documentation and automate the compilation.

Reply:
In addition, we should encourage the work streams to work collaboratively on changes, as we usually do.  Hence, people are aware of changes upfront, and often can discuss these proposed changes in calls.

Reply:
In RM and RA-1 we have also feedback from outside (great spelling and grammar fixes from GSMA people who can use spellchecker, which we lack).
To go through them in GitHub again, with 4 - 5 people in the current PR process, just for the sake of the process is just plainly silly.  As we all seem to agree.  Thanks to everyone!

Reply:
I am happy to approve this text in the current PR, if the new process is stated like that.

Reply:
I am fine with the changes (and edits by @pgoyal01 ) as well as support @gkunz 's proposal to remove the requirement on the reviewer's company for these minor changes.

Reply:
Requirement removed in [363dd87](https://github.com/cntt-n/CNTT/pull/3187/commits/363dd87b30bebc49341be6fba709831a7b3a5dcd)